src/Ng/Scripts/Catalog2DnxV3Secondary.cmd

```diff
@@ -0,0 +1,14 @@
+@echo OFF
+	
+cd Ng
+
+:Top
+	echo "Starting job - #{Jobs.ngcatalog2dnx.Title.Secondary}"
+
+	title #{Jobs.ngcatalog2dnx.Title.Secondary}
```

Comment:
Do you need to add these values to the Octopus variable set too or all of these already in there? I would make a note somewhere so that these values aren't missed if so during deployments.